features/support/hooks.rb

```diff
@@ -1,2 +1,2 @@
 # -*- encoding : utf-8 -*-
-Before do; end
+Before { ; }
```

Comment:
Yup. It's a block with a single empty statement.
Not sure if i prefer the { ; } syntax tbh, i just cargo-culted it from the styeguide.